.circleci/config.yml

```diff
@@ -50,7 +50,7 @@ jobs:
           command: composer install -n --prefer-dist
       - run:
           name: Run unit tests
-          command: ./vendor/bin/codecept run unit --xml unit_test_report.xml --coverage
+          command: ./vendor/bin/codecept run --xml unit_test_report.xml --coverage
```

Comment:
should the report still be called unit though?

Reply:
Probably not.

```suggestion
          command: ./vendor/bin/codecept run --xml test_report.xml --coverage
```